### ISSUE TITLE
[FE] Side Menu

### ISSUE THREAD

**Issue by Evanlab02** [LittleClumsy/Lyzer#33]
## Description

Please view the goal for this: #12 

As a user, when I visit Lyzer, I should be able to navigate across the app using a side menu.

### Acceptance Criteria

AC1 - Should have a menu item for the overview page.
AC2 - Should be collapsible.
AC3 - Allows opening with menu button on the navigation bar.

## Reference

![Image](https://github.com/user-attachments/assets/d0246307-67ae-4f2b-8d7b-fc3fe06b9cb5)

![Image](https://github.com/user-attachments/assets/008a719b-bf5a-4b98-b684-c4b94dd3db87)

**Comment by Evanlab02** [comment]
@Pranav-Zodes-44 I do not think we **need** to have like the little mini sidebar if it is too much of a hassle.
One that is completely closed IMO is fine. Just thought I would put it out there

(Definitely speaking out of pain from the last time)

![Image](https://github.com/user-attachments/assets/4f9d18ce-b7b4-4d4f-a2a5-9bcf20c00046)

**Comment by Pranav-Zodes-44** [comment]
PTSD is real 😂 Happy to keep it closed
